tracer/test/Directory.Build.props

```diff
@@ -19,29 +19,18 @@
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
   </ItemGroup>
 
+  <!-- This property ensures NuGet packages are copied to the bin folder. This happens automatically in netcoreapp3.0+ -->
+  <PropertyGroup Condition=" $(DD_LOGGER_ENABLED) != 'false' AND '$(TargetFramework)' == 'netcoreapp2.1' ">
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
   <ItemGroup Condition=" $(DD_LOGGER_ENABLED) != 'false' ">
-    <PackageReference Include="DatadogTestLogger" Version="0.0.23">
-      <CopyToOutputDirectory>lib\$(TargetFramework)\*</CopyToOutputDirectory>
-      <FolderName>datadogtestlogger</FolderName>
-    </PackageReference>
+    <PackageReference Include="DatadogTestLogger" Version="0.0.25" ExcludeAssets="compile"/>
```

Comment:
Where is the source for `DatadogTestLogger`? I couldn't find it anywhere

Reply:
https://github.com/tonyredondo/datadog-test-logger

Reply:
I can prepare a transfer to the Datadog org if we want that...

Reply:
I think that makes sense tbh, probably both the repo + the NuGet